Azure.ResourceManager.Core/AzureResourceManagerClient.cs

```diff
@@ -164,17 +164,17 @@ public IEnumerable<Location> ListAvailableLocations(string subscription, Resourc
                 .Locations.Cast<Location>();
         }
 
-        public ResourceGroupOperations ResourceGroup(string subscription, string resourceGroup)
+        public ResourceGroupOperations GetResourceGroupOperations(string subscription, string resourceGroup)
         {
             return new ResourceGroupOperations(ClientOptions, $"/subscriptions/{subscription}/resourceGroups/{resourceGroup}");
         }
 
-        public ResourceGroupOperations ResourceGroup(ResourceIdentifier resourceGroup)
+        public ResourceGroupOperations GetResourceGroupOperations(ResourceIdentifier resourceGroup)
         {
             return new ResourceGroupOperations(ClientOptions, resourceGroup);
         }
 
-        public ResourceGroupOperations ResourceGroup(ResourceGroupData resourceGroup)
+        public ResourceGroupOperations GetResourceGroupOperations(ResourceGroupData resourceGroup)
```

Comment:
return ResourceGroup
```suggestion
        public ResourceGroup GetResourceGroup(ResourceGroupData resourceGroup)
```